core/services/ocr2/plugins/functions/integration_tests/v1/internal/testutils.go

```diff
@@ -428,6 +428,10 @@ func AddBootstrapJob(t *testing.T, app *cltest.TestApplication, contractAddress
 		[relayConfig]
 		chainID                           = 1337
 		fromBlock                         = 1
+		donId                             = "1"
+		contractVersion                   = 1
+		contractUpdateCheckFrequencySec   = 1
```

Comment:
Is there precedent for `Sec` as a suffix?

Reply:
A precedent of multiple Functions' fields, yes. I personally find it more obvious to have a suffix in the name and use a plain int rather than a string like "1s". If that's inadvisable then we can discuss and address it in a separate refactor PR. Here the filed needs to match the one in Functions config.